src/main/src/main/java/applicationContext.xml

```diff
@@ -16,6 +16,9 @@
     </bean>
     <bean id="lockProviderInitializer" class="org.geoserver.config.LockProviderInitializer"/>
     
+    
+    <bean id="resourceWatcher" class="org.geoserver.platform.resource.ResourceWatcher"/>
```

Comment:
I don't think this is the right configuration for resourceWatcher, since `org.geoserver.platform.resource.ResourceWatcher` is an interface. Is this meant to be `org.geoserver.platform.resource.SimpleResourceWatcher`?